Engine/Results/LiveTradingResultHandler.cs

```diff
@@ -1045,7 +1045,7 @@ public void ProcessSynchronousEvents(bool forceProcess = false)
                             else
                             {
                                 // we haven't gotten data yet so just spoof a tick to push through the system to start with
-                                security.SetMarketPrice(new Tick(DateTime.Now, subscription.Configuration.Symbol, price, price));
+                                security.SetMarketPrice(new Tick(time, subscription.Configuration.Symbol, price, price));
```

Comment:
Thanks, I traced it through & its not used anywhere (likely why it was DateTime.Now before).